### PR TITLE
Workspace - GeoServer layers

### PR DESCRIPTION
**Pull request by Piotrek605**

### CODE REVIEW

**Comment by Piotrek605** [comment]
Description: changed the layer retrieval protocol to WMS so that layers are requested from GeoServer.

File: meteor-app/imports/ui/pages/workspace/component.js [hunk 1]
```diff
@@ -149,62 +149,60 @@ export default class WorkspacePage extends React.Component {
...
+                    server-type="geoserver"
                   />
                   {!layer.nextUrl ? null : (
                     <map-layer-xyz
                       name={`${layer.name} (preload)`}
                       url={layer.nextUrl}
                       min-zoom={layer.minZoom}
                       max-zoom={layer.maxZoom}
-                      opacity="0"
+                      opacity={layerOpacity}
```
Comment (Zodiase) [inline meteor-app/imports/ui/pages/workspace/component.js]: This is nowhere defined. Also this preloading layer is supposed to be never visible.

File: meteor-app/imports/ui/reducers/initialState.js [hunk 2]
```diff
@@ -14,23 +14,37 @@ export default {
     //! For testing only. Should be empty on start.
     layers: [
       {
-        name: 'Water-year Precipitation',
-        urlTile: 'PPT_water_year',
```
Comment (Zodiase) [inline meteor-app/imports/ui/reducers/initialState.js]: The `urlTile` is used as ID for the layer, while the `name` is purely used for displaying purposes. Please don't mix them together.

File: meteor-app/imports/ui/pages/workspace/component.js [hunk 3]
```diff
@@ -149,62 +149,60 @@ export default class WorkspacePage extends React.Component {
...
+                <div>
+                  <label>Opacity: </label>
+                  <input type="range" min="0" max="255" step="1" value={layer.opacity * 255} data-layer-index={layerIndex} onChange={this._bound_layerOpacityOnChange} />
+                  <label>{layer.opacity.toFixed(2)}</label>
+                </div>
+              </li>
+            ))}
+          </ul>
           <div className="section_map">
-            <ul className="layer-list">
```
Comment (Zodiase) [inline meteor-app/imports/ui/pages/workspace/component.js]: Seems like a unnecessary change here.

Comment (Piotrek605) [inline meteor-app/imports/ui/pages/workspace/component.js]: You need to be more specific in those comments and what should be changed. Do you mean layer-list being outside section_map instead of inside? It doesn't matter here but in the final layout the slider and the layer list make up one section and the map is a different section.

Comment (Zodiase) [inline meteor-app/imports/ui/pages/workspace/component.js]: Depending on the merge order, this may or may not cause a conflict. My bet is on it will. Also unnecessary changes are unnecessary changes. You don't add a piece of irrelevant change in a PR and say "I'm going to use it later for another thing". Just don't do that.
